test/services/postcard.spec.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Config  = require('../../config');
```

Comment:
nit: alpha order

also we normally separate imports from external packages and code from the same repo into different sections separated by a blank line, so you can move `var Config  = require('../../config');` below these